src/core/prompts/instructions/create-mcp-server.ts

```diff
@@ -50,6 +50,7 @@ Common configuration options for both types:
 - \`disabled\`: (optional) Set to true to temporarily disable the server
 - \`timeout\`: (optional) Maximum time in seconds to wait for server responses (default: 60)
 - \`alwaysAllow\`: (optional) Array of tool names that don't require user confirmation
+- \`disabledForPromptTools\`: (optional) Array of tool names that are not included in the system prompt and won't be used
```

Comment:
Naming is hard and I understand how we ended up here... but IMO `disabledTools` would be a cleaner choice.